README.md

```diff
@@ -5,6 +5,8 @@
 [![Devops Build Status](https://dev.azure.com/typescript/TypeScript/_apis/build/status/Typescript/node10)](https://dev.azure.com/typescript/TypeScript/_build?definitionId=7)
 [![npm version](https://badge.fury.io/js/typescript.svg)](https://www.npmjs.com/package/typescript)
 [![Downloads](https://img.shields.io/npm/dm/typescript.svg)](https://www.npmjs.com/package/typescript)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/microsoft/TypeScript/badge)](https://api.securityscorecards.dev/projects/github.com/microsoft/TypeScript)
+

```

Comment:
I actually think we should just remove all these links. Thoughts @RyanCavanaugh?

Reply:
Selfishly I do like the CI ones just because you can't click the red X on commits on mobile, so it's the only way to see what probably failed.